src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs

```diff
@@ -621,13 +348,13 @@ private static int WriteHex(int value, Span<char> destination, int written)
         }
 
         /// <summary>
-        /// Converts a number 0 - 15 to its associated hex character '0' - 'f' as byte.
+        /// Converts a number 0 - 15 to its associated hex character '0' - 'F' as byte.
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static byte Int32LsbToHexDigit(int value)
         {
             Debug.Assert(value < 16);
-            return (byte)((value < 10) ? ('0' + value) : ('a' + (value - 10)));
+            return (byte)((value < 10) ? ('0' + value) : ('A' + (value - 10)));
```

Comment:
What motivated this change? Consistency with `JavascriptEncoder`?

Reply:
Yes. Not related to this issue